internal/service/federatedsettingsorgconfig/resource_federated_settings_connected_org.go

```diff
@@ -85,10 +97,19 @@ func resourceMongoDBAtlasFederatedSettingsOrganizationConfigRead(ctx context.Con
 	if err := d.Set("domain_allow_list", federatedSettingsConnectedOrganization.DomainAllowList); err != nil {
 		return diag.FromErr(fmt.Errorf("error setting domain allow list (%s): %s", d.Id(), err))
 	}
+	if err := d.Set("data_access_identity_provider_ids", federatedSettingsConnectedOrganization.GetDataAccessIdentityProviderIds()); err != nil {
+		return diag.FromErr(fmt.Errorf("error setting data_access_identity_provider_ids (%s): %s", d.Id(), err))
+	}
 
 	if err := d.Set("post_auth_role_grants", federatedSettingsConnectedOrganization.PostAuthRoleGrants); err != nil {
 		return diag.FromErr(fmt.Errorf("error setting post_auth_role_grants (%s): %s", d.Id(), err))
 	}
+	if err := d.Set("user_conflicts", FlattenUserConflicts(federatedSettingsConnectedOrganization.GetUserConflicts())); err != nil {
+		return diag.FromErr(fmt.Errorf("error setting `user_conflicts` for federatedSettings IdentityProviders: %s", err))
```

Comment:
knit: use consistently Errorf or FromError.      Errorf seems better, we might want to have a ticket to unify use in all resources